From 08e1e8bcc5a8e1393894ad04abdad25c94bfc1c7 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 9 Jan 2025 18:08:18 +0100
Subject: [PATCH 1442/1695] drivers: stm32_bsec3: remove duplicate macros

Some macros for BSEC_SR register fields are defined twice. Remove the
extra ones.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1ddfb28710ea06f0c484482c255b640a5f46d062 drivers: stm32_bsec3: remove duplicate macros]
Change-Id: Ib59dad317fffa30b37ed8ef69c50a18d58dfb6b8
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/427936
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/drivers/stm32_bsec3.c | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/core/drivers/stm32_bsec3.c b/core/drivers/stm32_bsec3.c
index 88013f727..5972caf5e 100644
--- a/core/drivers/stm32_bsec3.c
+++ b/core/drivers/stm32_bsec3.c
@@ -43,14 +43,6 @@ static_assert(!(IS_ENABLED(CFG_STM32_CM33TDCID) &&
 #define BSEC_VERR			U(0xFF4)
 #define BSEC_IPIDR			U(0xFF8)
 
-/* BSEC_SR register fields */
-#define BSEC_SR_BUSY			BIT(0)
-#define BSEC_SR_HVALID			BIT(1)
-#define BSEC_SR_STATE_MASK		GENMASK_32(31, 26)
-#define BSEC_SR_STATE_SHIFT		26U
-#define BSEC_SR_STATE_OPEN		U(0x16)
-#define BSEC_SR_STATE_CLOSED		U(0x0D)
-
 /* BSEC_OTPCR register fields */
 #define BSEC_OTPCR_PROG			BIT(13)
 #define BSEC_OTPCR_PPLOCK		BIT(14)
@@ -124,6 +116,7 @@ static_assert(!(IS_ENABLED(CFG_STM32_CM33TDCID) &&
 #define BSEC_DENR_NI		BSEC_DENR_v(0x0fdf)
 
 /* BSEC_SR register fields */
+#define BSEC_SR_BUSY			BIT(0)
 #define BSEC_SR_HVALID			BIT(1)
 #define BSEC_SR_NVSTATES_MASK		GENMASK_32(31, 26)
 #define BSEC_SR_NVSTATES_SHIFT		26U
-- 
2.47.2

